osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSubScreen.cs

```diff
@@ -321,6 +323,26 @@ private void onUserModsChanged(ValueChangedEvent<IReadOnlyList<Mod>> mods)
             client.ChangeUserMods(mods.NewValue);
         }
 
+        private void updateBeatmapAvailability(ValueChangedEvent<BeatmapAvailability> _ = null)
+        {
+            if (client.Room == null)
+                return;
+
+            client.ChangeBeatmapAvailability(BeatmapAvailability.Value);
+
+            if (client.LocalUser?.State == MultiplayerUserState.Ready)
```

Comment:
shouldn't this be handled server side..?

Reply:
this is indeed handled server side (although the resetting of server side beatmap availability is non-existent - it should be resetting it on beatmap change).

Reply:
but I believe we've decided that beatmap availability should only ever be for display purposes? either way I don't mind.

Reply:
yes, it's for display purposes. but the display is wrong right now. because it's not being updated when the beatmap changes server side, so will **display** wrong for a period.